earthnet_models_pytorch/model/local_rnn.py

```diff
@@ -142,11 +144,12 @@ def forward(self, data, pred_start: int = 0, n_preds: Optional[int] = None):
 
         else:
 
-            meso_dynamic_inputs = meso_dynamic_inputs.reshape(b*t_m, c_m, h_m, w_m)
+            meso_dynamic_inputs = meso_dynamic_inputs.reshape(b*t_m, c_m, 1, 1) #h_m, w_m)
```

Comment:
we should add this somehow that it both works for en21 (with mesoscale weather) and for en23